procfs/cmdline.go

```diff
@@ -184,6 +184,40 @@ func (c *Cmdline) SetAll(args []string) {
 	}
 }
 
+// Delete deletes a kernel parameter by exact match.
+func (c *Cmdline) Delete(v *Parameter) {
+	c.Lock()
+	defer c.Unlock()
+
+	for i, value := range c.Parameters {
+		if value.key == v.key {
+			for _, val := range v.values {
```

Comment:
if we drop all values, should we drop the outer key?

I mean like we had

`console=abc`

and I append with `-console=abc`, should it drop `console` key completely now?

Reply:
i didn't quite get that, this maybe: https://github.com/siderolabs/go-procfs/pull/11/files#diff-1f6502d8a4e4009fba4aa162aba1cb59dca996f3dee72949ac177e56fc0185b0R136-R138

Reply:
I guess it's fine